exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/TableBuilder.java

```diff
@@ -56,6 +47,30 @@ public TableBuilder(final String[] columns) {
     sb.append("</tr>\n");
   }
 
+  /**
+   * Representation of a millisecond duration in a short readable text
+   * @param millis  Duration in milliseconds
+   * @return        Human-Readable Duration Time
+   */
+  public String shortDurationFormat(long millis) {
+    long days = TimeUnit.MILLISECONDS.toDays(millis);
+    long hours = TimeUnit.MILLISECONDS.toHours(millis) - TimeUnit.DAYS.toHours(TimeUnit.MILLISECONDS.toDays(millis));
+    long minutes = TimeUnit.MILLISECONDS.toMinutes(millis) - TimeUnit.HOURS.toMinutes(TimeUnit.MILLISECONDS.toHours(millis));
+    long seconds = TimeUnit.MILLISECONDS.toSeconds(millis) - TimeUnit.MINUTES.toSeconds(TimeUnit.MILLISECONDS.toMinutes(millis));
+    long milliSeconds = millis - TimeUnit.SECONDS.toMillis(TimeUnit.MILLISECONDS.toSeconds(millis));
+    String formattedDuration = "";
+    if (days >= 1) {
+      formattedDuration = days + "d" + hours + "h" + minutes + "m";
+    } else if (hours >= 1) {
+      formattedDuration = hours + "h" + minutes + "m";
+    } else if (minutes >= 1) {
+      formattedDuration = minutes + "m" + seconds + "s";
+    } else {
+      formattedDuration = seconds + "." + milliSeconds + "s";
+    }
+    return formattedDuration;
```

Comment:
A nit, but the code is clearer if you do:

```
if (cond1) {
    return expr1;
} else if (cond2) {
    return expr2;
} else {
    return expr3;
}
```

The above eliminates the temp variable and makes it very clear what is happening.